plugins/polio/api/perfomance_dashboard/permissions.py

```diff
@@ -28,12 +28,9 @@ def has_permission(self, request, view):
         - Read access for safe methods if user has any of the required permissions.
         - Write access for unsafe methods if user has write or admin permissions.
         """
+
         if request.method in permissions.SAFE_METHODS:
-            return (
-                request.user.has_perm(POLIO_PERFORMANCE_READ_ONLY_PERMISSION.full_name())
-                or request.user.has_perm(POLIO_PERFORMANCE_NON_ADMIN_PERMISSION.full_name())
-                or request.user.has_perm(POLIO_PERFORMANCE_ADMIN_PERMISSION.full_name())
-            )
+            return True
```

Comment:
@Crebert08  What you did was good. I just changed it to make the endpoints publicly accessible for read-only operations. That will make things easier if/when we need to embed the related pages